target/bin/setup-fetchmail

```diff
@@ -1,13 +1,14 @@
-#! /bin/sh
+#! /bin/bash
```

Comment:
Strictly speaking it is still not the identical shebang in all scripts. some have a space after the pound.

Reply:
Fixed in the latest commit.